addon/components/session-copy.hbs

```diff
@@ -1,21 +1,22 @@
 <div
   class="session-copy"
   {{did-insert (perform this.setup) @session}}
+  {{did-insert this.scrollIntoView}}
```

Comment:
Ugh, bad initial suggestion, sorry. We actually have a modifier that does this. You can just do `{{scroll-into-view}}` and it should handle it.

Reply:
> Ugh, bad initial suggestion, sorry. We actually have a modifier that does this. You can just do `{{scroll-into-view}}` and it should handle it.

haha, no prob. done. and good to know!